packages/commonwealth/client/scripts/views/modals/edit_topic_thresholds_modal.tsx

```diff
@@ -70,23 +73,25 @@ type EditTopicThresholdsModalProps = {
   onModalClose: () => void;
 };
 
-export const EditTopicThresholdsModal = (
-  props: EditTopicThresholdsModalProps
-) => {
+export const EditTopicThresholdsModal = ({
+  onModalClose,
+}: EditTopicThresholdsModalProps) => {
+  const { data: topics } = useFetchTopicsQuery({
```

Comment:
this can be compared to our old way of getting data from state 
```const topics = app.topics.getByCommunity(app.activeChainId());```

Now we use hook and it reads it from the cache immediately. Then, if the data is stale, it hits API and update cache with fresh data. If data is not stale, API call is ommited. That is why it is important to handle `staleTime` properly and correctly invalidate cache